daemon/tests/harness/mod.rs

```diff
@@ -367,9 +367,13 @@ impl Taker {
 #[macro_export]
 macro_rules! deliver_event {
     ($maker:expr, $taker:expr, $event:expr) => {
-        tracing::debug!("Delivering event: {:?}", $event);
-        $taker.system.cfd_actor_addr.send($event).await.unwrap();
-        $maker.system.cfd_actor_addr.send($event).await.unwrap();
+        #[allow(unused_must_use)]
```

Comment:
Why? Shouldn't we `unwrap` here?

Reply:
Unfortunately we cannot `unwrap` twice unconditionally, as sometimes sending only returns `Result<(), Disconnected>`.

Reply:
Ah that is a shame. @klochowicz are you okay with this for now? You introduced the macro but unfortunately, it is no longer as generically applicable. We might need two macros (`deliver_fallible_event` and `deliver_event`) or create some other abstraction on the actual `Maker` and `Taker` structs.